CIP-????/README.md

```diff
@@ -0,0 +1,211 @@
+|Alonzo|Plutus scripts|[spec](https://github.com/input-output-hk/cardano-ledger/releases/latest/download/alonzo-ledger.pdf)|
+|Babbage|improved Plutus script contexts|[spec](https://github.com/input-output-hk/cardano-ledger/releases/latest/download/babbage-ledger.pdf)|
+|Conway|governance|[spec WIP](https://github.com/input-output-hk/formal-ledger-specifications)|
+
+Note that there is no Allegra specification.
+The Allegra era consists entirely of the addition of timelocks to the MultiSig script
+introduced in the Shelley ledger era
+(See figure 12 of the Mary specification).
+
+Note that small, isolated changes can be made within a ledger era
+by way of an intra-era hard fork. See [CIP-59] for more details.
+
+#### Ledger events
+
+Some provenance about the ledger calculations is provided by ledger events.
+Sometimes these events have clear triggers (e.g. Plutus script execution)
+and sometimes they provide intermediate calculations performed by the ledger rules
+(e.g. the reweard calculation).
+The events come with zero cost to a running node if not used and are not stored in the ledger state.
+Documentation about the existing events can be found
+[here](https://github.com/input-output-hk/cardano-ledger/blob/master/docs/LedgerEvents.md).
+
+### Soft forks and Hard forks
+
+Since most ledger CIPs will involve backwards incompatible changes,
+the following two definitions are helpful:
+
+**Hard fork** - A *hard fork* is a change to the protocol (not restricted to the ledger)
+resulting in a single new block definition becoming valid.
+
+Alternatively, a hard fork is a backwards incompatible change for both
+block producers and block validators.
+
+**Soft fork** - A *soft fork* is a change to the protocol (not restricted to the ledger)
+resulting in fewer blocks being valid.
+
+Alternatively, a soft fork is a backwards incompatible change for
+block producers, but a backwards compatible change for block validators.
+
+### Serialization
+
+Transactions and blocks are serialized with
+[CBOR](https://www.rfc-editor.org/rfc/rfc7049)
+and specified with
+[CDDL file](https://www.rfc-editor.org/rfc/rfc8610).
+
+Serialization changes to the ledger are discussed in
+[CIP-80](https://github.com/cardano-foundation/CIPs/pull/372).
+
+Note that the serialisation format of the ledger state is unspecified and left as an
+implementation detail (unlike the format of blocks).
+
+### The ledger-script interface
+
+The ledger and Plutus scripts have a common interface, described in [CIP-35].
+CIPs relating to this inteface are relevant to both the ledger and to the Plutus CIP categories.
+
+Additionally, there is significant overlap with the Ledger category around the ledger-script
+interface and the protocol parameters.
+CIPs which change these parts of Cardano should generally use the Plutus category and not the
+Ledger category, although the Editors may ask the Ledger reviewers to comment.
+
+## Specification
```

Comment:
I see that CIP-35 has a [Reviewers](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0035#the-plutus-reviewers) section now. I hadn't planned on adding a table like that, I suspect that the folks I would put in such a table would end up review ledger CIPs anyway. Should I re-consider? cc @michaelpj

Reply:
I'm not sure. I put it there because the new CIP-001 somewhat implicitly has the concept of project maintainers/reviewers who are the people who get called upon to review CIPs in that category, and I thought it was better to make it explicit who that was.